salt/orchestrate/aws/mitxpro_autoscaling.sls

```diff
@@ -9,18 +13,14 @@
 {% set sqs_queue = env_data.provider_services[app_name].sqs.queue ~ '-' ~ ENVIRONMENT ~ '-autoscaling' %}
 {% set sns_topic = env_data.provider_services[app_name].sns.topic ~ '-' ~ ENVIRONMENT ~ '-autoscaling' %}
 {% set edx_codename = purpose_data.versions.codename %}
-{% if app_name == 'edxapp' %}
-{% set security_groups = ['edx'] %}
-{% else %}
-{% set security_groups = ['edx-worker'] %}
-{% endif %}
-{% do security_groups.extend(['master-ssh', 'consul-agent', 'default']) %}
+{% set env_suffix = ENVIRONMENT.strip('-')[1] %}
+{% set security_groups = ['{}-salt-minion'.format(ENVIRONMENT), '{}-consul-agent'.format(ENVIRONMENT), 'default', 'mitxpro-edxapp-access-{}'.format(env_suffix), 'mitxpro-edxapp-db-access-{}'.format(env_suffix)]) %}
```

Comment:
We don't want to assign the db security group to the edx instances. Also, might be good to split this across multiple lines for readability.